index.html

```diff
@@ -116,7 +117,7 @@ <h3>Shop</h3>
             
         </div>
         <div class="information-button">
-            <a href="">Purchase tickets</a>
+            <a href="">Purchase Tickets</a>
         </div>
       </section>
       <section class="band" id="band">
```

Comment:
- [ ] You did a very good job on the project but for this project, you are supposed to stick to the [Behance ](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) design as much as possible. When we hover the mouse over cards a white border around cards should appear as on the design, Please apply hover effects to your cards so the cards show a white border when we hover the mouse over them.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/157689981-91c089cf-d233-49a4-b0d5-3e74f7a6d17b.png)